server/xpub-controller/helpers/files.js

```diff
@@ -0,0 +1,151 @@
+const logger = require('@pubsweet/logger')
+const FileModel = require('@elifesciences/xpub-model').File
+
+class FilesHelper {
```

Comment:
Maybe worth a comment on the class that although this is a generic "FilesHelper" it is very specific to MANUSCRIPT_SOURCE ... in the future it would be good to handle the supplementary files too.